x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/SparseVectorFieldMapper.java

```diff
@@ -142,6 +147,8 @@ public SparseVectorFieldType fieldType() {
 
     @Override
     public void parse(ParseContext context) throws IOException {
+        deprecationLogger.deprecatedAndMaybeLog("sparse_vector", DEPRECATION_MESSAGE);
+
```

Comment:
should we issue a deprecation logging only when parsing a sparse_vector mapping in `public Mapper.Builder<?,?> parse(..)` instead of issuing it every time while parsing a document with a sparse_vector?

Reply:
Yes, sorry I asked you to review a bit too early :) I still need to fix this, plus some failing docs tests.